packages/docusaurus-theme-classic/src/theme/Layout/index.tsx

```diff
@@ -32,7 +32,9 @@ function Layout(props: Props): JSX.Element {
 
       <Navbar />
 
-      <div className={clsx('main-wrapper', wrapperClassName)}>{children}</div>
+      <div className={clsx('main-wrapper', wrapperClassName, pageClassName)}>
```

Comment:
Maybe add `main-wrapperr` to the theme classnames too? like `ThemeClassNames.wrappers.allPages` ?

Reply:
I think `ThemeClassNames.wrappers.main` would be better